MyOwnList/src/MyList.java

```diff
@@ -0,0 +1,42 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class MyList implements OwnList {
+
+    List<Integer> MyList = new ArrayList<>();
```

Comment:
eee, co to panie :D Lista zbudowana na liście? Założ, że lista nie istnieje ;) Tutaj powinna być tablica.

btw. 'MyList' - od kiedy to tak piszemy zmienne z wielkiej litery?